com.skype.Client.appdata.xml

```diff
@@ -22,5 +22,6 @@
   <icon type="remote" height="128" width="128">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w128</icon>
   <releases>
     <release date="2018-01-07" version="8.13.76.8"/>
+    <release date="2018-02-01" version="8.15.76.2"/>
```

Comment:
I think this has caused confusion before where order mattered.. it is probably easier to just replace the old one since it has no release description anyway.

Reply:
Ok, so should I just point the current version every time then?

Reply:
Yes that will be fine.